scripts/yarn

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${IOW_BOUNDARY_TOOL_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Execute Yarn CLI commands.
```

Comment:
No preference on yarn vs npm from me, but just curious of an example command we would be using this script for.

Reply:
This is useful when adding dependencies. Instead of doing:

```bash
docker-compose run --rm --no-deps app yarn add leaflet react-leaflet
```

We can do:

```bash
./scripts/yarn add leaflet react-leaflet
```

Reply:
That makes sense. I think in the past I have just updated the `package.json` file directly and then ran `./scripts/update`, but I think editing the package.json directly is kind of frowned upon.